examples/showcase/ui/info-panel.js

```diff
@@ -4,7 +4,7 @@ AFRAME.registerComponent('info-panel', {
     var buttonEls = document.querySelectorAll('.menu-button');
     var fadeBackgroundEl = this.fadeBackgroundEl = document.querySelector('#fadeBackground');
 
-    this.movieImageEl;
```

Comment:
we can probably remove this line all together

Reply:
Sure - have done so.